### PR TITLE
DHFPROD-4771: Update Load tile

### PR DESCRIPTION
**Pull request by wooldridge**
Includes:
- removing Files information from Load cards
- rewriting switch view component, using MLRadio component from Design System
- adding tests for LoadData component
- renaming page component names to match UI names (Curate, Load, Run)

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/entities/matching/matching-card.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const MatchingCard: React.FC<Props> = (props) => {
         </Modal>;
 
     return (
-        <div className={styles.loaddataContainer}>
+        <div className={styles.loadContainer}>
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/components/entities/matching/matching-card.tsx:101]: Should this still be loadContainer, even though it's for the match card? Perhaps a more general name - these are all steps, so "stepContainer"?

File: marklogic-data-hub-central/ui/src/pages/Run.tsx [hunk 2]
```diff
@@ -20,8 +20,8 @@ const Statuses = {
     'FINISHED_WITH_ERRORS': 'finished_with_errors'
 }
 
-// TODO Rename Bench component to Run
-const Bench: React.FC = () => {
+// TODO Rename Run component to Run
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/pages/Run.tsx:23]: Can remove this comment?

Comment (wooldridge) [reply marklogic-data-hub-central/ui/src/pages/Run.tsx]: Thanks, @rjrudin.

File: marklogic-data-hub-central/ui/src/components/entities/matching/matching-card.tsx (hunk 1 above)
Comment (wooldridge) [reply marklogic-data-hub-central/ui/src/components/entities/matching/matching-card.tsx]: Thanks, @rjrudin.

**Comment by bsrikan** [comment]
Can the reviewers mark "Reviewed tests" if you have reviewed that the tests in this PR cover what is listed in the test plan. We shouldnt be approving if it is not the case.

**Comment by wooldridge** [comment]
Here is the test plan: https://wiki.marklogic.com/display/ENGINEERING/Test+Plan+for+Unified+Experience+in+HC

**Pull request merged**